core/model/modx/modresource.class.php

```diff
@@ -450,6 +450,41 @@ function __construct(xPDO & $xpdo) {
         $this->_cacheKey= "[contextKey]/resources/[id]";
     }
 
+    /**
+     * Compute the "preview URL" for the resource
+     *
+     * @return string
+     */
+    public function getPreviewUrl() {
+        if ($this->get('deleted')) {
+            return '';
```

Comment:
This is not related to this "issue" per se, but i'm just wondering why one can't preview a deleted resource (granted the permission is given)

Reply:
A deleted resource will result in a 404 when requested.